docs/gluster-block.8

```diff
@@ -102,8 +103,16 @@ show help message and exit.
 show version info and exit.
 .PP
 
-.SH SUPPORTED JSON-FORMATS
---json | --json-plain | --json-spaced | --json-pretty
+.SH COMMON CLI OPTIONS
+.SS
+[\fBtimeout <seconds>\fR]
+it is the time in seconds that cli can wait for daemon to respond (default: 300)
+.PP
+
+.SS
+[\fB--json*\fR]
+used to request the output result in json format (default: plain text) and supported JSON formats: --json|--json-plain|--json-spaced|--json-pretty
+.PP
 
 .SH EXAMPLES
```

Comment:
@lxbsz can you also correct:
To create a block device of size 1GiB, by preallocating storage with zero fill
.B # gluster-block create blockVol/sampleBlock prealloc full ${HOST} 1GiB

to

Some meaning ful msg here
.B # gluster-block create blockVol/sampleBlock prealloc no ${HOST} 1GiB

And also add an example for timeout option usage.

Thanks!

Reply:
Yeah, will fix it.
Thanks.